### PR TITLE
Add pointer events test for pointerId across frames

### LINKED ISSUE #3130: Add pointerevent pointerId scope test
We need a test to make sure the pointerId of an active pointer is the same across iframes

### PR DESCRIPTION
**Pull request by NavidZ**
closes #3130

### CODE REVIEW

**Comment by NavidZ** [comment]
I was not sure how to exclude the iframe html page the resources folder I added to some exclusion list so it is not considered a test by itself. I saw some iframe pages inside MANIFEST.json but some others weren't. Let me know if you know how I should do that if it is needed at all.

**Comment by NavidZ** [comment]
@jacobrossi @scottgonzalez @RByers, can you have a quick look at this new pointer event test?

File: pointerevents/pointerevent_pointerId_scope-manual.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                var pointerover_pointerType = null;
+
+                var eventList = ['pointerenter', 'pointerover', 'pointermove', 'pointerout', 'pointerleave'];
+                var receivedEvents = {};
+                var receivedEventsInnerFrame = {};
+
+
+                function checkPointerId(event, inner) {
+                    detected_pointertypes[event.pointerType] = true;
+                    var eventName = (inner ? "inner frame " : " ") + event.type;
```
Comment (scottgonzalez) [inline pointerevents/pointerevent_pointerId_scope-manual.html]: Why a space instead of an empty string?

File: pointerevents/pointerevent_pointerId_scope-manual.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                    var pe_event = JSON.parse(event.data);
+                    receivedEventsInnerFrame[pe_event.type] = 1;
+                    checkPointerId(pe_event, true);
+                    if (Object.keys(receivedEvents).length == eventList.length && Object.keys(receivedEventsInnerFrame).length == eventList.length)
+                        test_pointerEvent.done();
+                });
+                
+                on_event(target0, 'gotpointercapture', function(event) {
+                     target0.releasePointerCapture(event.pointerId);
+                });
```
Comment (scottgonzalez) [inline pointerevents/pointerevent_pointerId_scope-manual.html]: Why is this needed?

Comment (NavidZ) [inline pointerevents/pointerevent_pointerId_scope-manual.html]: I added it to pass it on Chrome as we do implicit capture for touches. However, now that you pointed it out, I remembered that it was not yet in the spec. So I'll remove this and if we change the spec to do any implicit capture I'll add this.

File: pointerevents/pointerevent_pointerId_scope-manual.html (hunk 1 above)
Comment (NavidZ) [inline pointerevents/pointerevent_pointerId_scope-manual.html]: Right. No need for that.

**Comment by scottgonzalez** [comment]
Looks good.

**Comment by RByers** [comment]
Sorry for the delay - missed this. 
One nit - can you rename "pointerevents/resources/pointerevent_pointerId_scope-manual.html" to avoid confusion with the main test with the same filename?  Maybe "pointerevent_pointerId_scope-iframe.html" or something?
